test/servlet31/faceletsEmptyAsNull/src/test/java/com/sun/faces/test/servlet31/faceletsemptyasnull/Issue2827IT.java

```diff
@@ -144,10 +144,10 @@ public void testValidateEmptyFields() throws Exception {
             assertEquals(integerInput.getValueAttribute(), "");
 
             pageAsText = page.asText();
-            assertTrue(pageAsText.contains("VC1 Fired: true"));
-            assertTrue(pageAsText.contains("VC2 Fired: true"));
-            assertTrue(pageAsText.contains("String model set with null: false"));
-            assertTrue(pageAsText.contains("Integer model set with null: false"));
+            assertTrue(pageAsText.contains("VC1 Fired: false"));
+            assertTrue(pageAsText.contains("VC2 Fired: false"));
+            assertTrue(pageAsText.contains("String model set with null: true"));
+            assertTrue(pageAsText.contains("Integer model set with null: true"));
```

Comment:
Was the original test wrong? This wasn't explained in conversation?

Reply:
I thought the test was wrong when I prepared https://github.com/eclipse-ee4j/mojarra/pull/4643, but I was wrong. This pull request reverts it back.